collector/Makefile

```diff
@@ -25,13 +28,19 @@ cmake-build/collector: $(HDRS) $(SRCS) txt-files $(shell find falcosecurity-libs
 		-e DISABLE_PROFILING="true" \
 		quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG) /build-collector.sh
```

Comment:
The docker run command should be `"$(SRC_MOUNT_DIR)/builder/build/build-collector.sh"`